core/serialization.go

```diff
@@ -393,6 +397,94 @@ func (h *BlobHeader) Deserialize(data []byte) (*BlobHeader, error) {
 	return h, err
 }
 
+// GetBatchHeader constructs a core.BatchHeader from a proto of pb.StoreChunksRequest.
+// Note the StoreChunksRequest is validated as soon as it enters the node gRPC
+// interface, see grpc.Server.validateStoreChunkRequest.
+func BatchHeaderFromProtobuf(in *pb.BatchHeader) (*BatchHeader, error) {
```

Comment:
nit: `BatchHeaderFromProto` to be consistent (or `BlobHeaderFromProtobuf` below if full spell is preferred)